src/pages/Dashboard/components/UserInformation/UserInformation.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, useTheme, /* styled, Paper, */ Grid, useMediaQuery, Typography } from '@mui/material'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import React from 'react'
+
+
+
+const UserInformation = (): JSX.Element => {
+
+    const theme = useTheme()
+    /* const Item = styled(Paper)({
+        padding: theme.spacing(1),
+        textAlign: 'center',
+        color: theme.palette.text.secondary,
+    }); */
+
+    const isMd = useMediaQuery(theme.breakpoints.up('md'));
+    return (
+        <Box sx={{ m: 2 }}>
+            User information / Number of appointments left.
+            <Box sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+            }}>
+                <Box sx={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    alignItems: 'center',
+                }}>
+                    <AccountCircleIcon sx={{ fontSize: isMd ? 250 : 200, color: 'primary' }} />
+                    <Typography mt={2} variant={isMd ? 'h3' : 'h4'}>
+                        Trevor Phillips
+                    </Typography>
+                </Box>
+                <Typography mt={4}>
+                    {/* available currently hardcoded */}
+                    <Typography color='primary' variant='h5' component='span'>3 </Typography>
+                    appointments available
```

Comment:
Please use translation. probably switch the order for the moment. So the number is in the right side.

for spanish:  "Citas restante:"